skyfield/starlib.py

```diff
@@ -40,7 +40,7 @@ class Star(object):
 
     def __init__(self, ra=None, dec=None, ra_hours=None, dec_degrees=None,
                  ra_mas_per_year=0.0, dec_mas_per_year=0.0,
-                 parallax_mas=0.0, radial_km_per_s=0.0, names=()):
+                 parallax_mas=0.0, radial_km_per_s=0.0, jd_of_position=T0, names=()):
```

Comment:
What would you think of naming this `epoch` to agree with the terminology of the Hipparcos catalog?

Reply:
I chose this version because it makes clear the units of the input value, in the same way the `_km_per_s` suffixes on some of the arguments to this function do.

Reply:
1. Maybe `epoch_jd`, then?
2. Drat, now I'm thinking of the much bigger question of whether this should take a `Time` object :) It doesn't need to, right? There's other places in the code already where the interface just sticks to a plain jd?

Reply:
(Just checking back in: before we land, do you have an opinion on the spelling here, and the question of allowing a Time vs just a float?)

Reply:
Thanks for the nudge. I don't like `epoch` as the root word of the name for this parameter since in my opinion it's more vague to the uninitiated — but I'm not sure how to make that choice anything more than a matter of opinion. In which case I think "package owner's opinion wins" is the only sensible resolution process ...

And, yeahhhh, if there are Time objects, it should probably take a Time object. It was sheer laziness on my part not to look into how to make that work. I'd presume it's pretty easy — just convert it to a JD and save that?

Reply:
Let's try `epoch` and I'll see what it looks like as I try using it after it lands! If they pass a float, then just keep it untouched; but if they pass a time object from Skyfield's `timelib`, I think you can ask it for its `.tt` attribute to get the float you need. Thanks!

Reply:
The `epoch` value is subtracted from a TDB time. Should I ask for the Time's `.tdb` attribute instead? (I am wondering if maybe asking for TDB opens up a can of worms.)

Reply:
Oh! Yes, then `.tdb` is the correct attribute, thanks!